docs/guides/networking/linode-network/ip-failover/index.md

```diff
@@ -21,33 +21,104 @@ There's always a small possibility that your Compute Instance may be powered off
+{{</note>}}
+
+1. Log in to the Cloud Manager.
+
+1. Determine which Compute Instances you wish configure for IP failover. They both must be located in the same data center. If you need to, create those Compute Instances now and allow them to boot up.
+
+1. Disable Network Helper on both Compute Instances. For instructions, see the [Network Helper](/docs/guides/network-helper/#single-per-linode) guide.
+
+1. Add an additional IPv4 address to one of the Compute Instances. See the [Managing IP Addresses](/docs/guides/managing-ip-addresses/#adding-an-ip-address) guide for instructions. Make a note of the new address that is assigned as this will be used as the shared IP address.
+
+1. On the other Compute Instance, configure the *IP Sharing* feature to use the IPv4 address that was just added to the other instance. See [Managing IP Addresses](/docs/guides/managing-ip-addresses/#configuring-ip-sharing) for instructions on configuring IP sharing.
+
+1. Configure IP failover on the internal system of *both* Compute Instances.
+
+    1.  Log in to the Compute Instance using [SSH](/docs/guides/connect-to-server-over-ssh/) or [Lish](/docs/guides/using-the-lish-console/).
+
+    1.  Review the networking configuration to ensure that the shared IP address is not listed.
+
+            ip addr
+
+        If the IP address *does* appear, remove it first with the following command:
+
+            ip addr del [shared-ip]/32 dev eth0
+
+        {{<caution>}}
+After the IP address has been removed, it is no longer accessible until BGP routing has been configured. Use caution if enabling IP Sharing on IPv4 addresses that are a part of a production workload as downtime is likely to occur.
+{{</caution>}}
+
+    1.  Add the shared ip address to the loopback interface:
+
+            ip addr add [shared-ip]/32 dev lo
+
+    1.  Download and install the [lelastic](https://github.com/linode/lelastic) utility from GitHub by running the following commands:
+
+            curl -LO https://github.com/linode/lelastic/releases/download/v0.0.3/lelastic.gz
+            gunzip lelastic.gz
+            chmod 755 lelastic
+            mv lelastic /usr/local/bin/
+
+    1.  Run the following command to configure BGP routing through lelastic, replacing *[id]* with the ID corresponding to your data center in the [table above](/docs/guides/ip-failover/#ip-failover-support) and *[role]* with either `primary` or `secondary`.
+
+            lelastic -dcid [id] -[role] &
+
+        Once configured, the shared IP address is routed to the primary system. If that system is inaccessible, it fails over to the secondary system *until* the primary system becomes available again. If both systems are configured as the same role (both primary or both secondary), then the behavior is slightly different. It still fails over to the other system should the original system become inaccessible, but then it remains routed to the other system, even if the original system comes back online.
```

Comment:
"It" at the beginning of the third sentence is weak here. Could be "the primary systems still fails over to the secondary system..."